challenges/deploy-WebV/README.md

```diff
@@ -0,0 +1,74 @@
+# Deploy WebValidate
+
+## Background
+
+Web Validate (WebV) is our end-to-end test tool that generates web requests and sends them to our NGSA application. These web requests can be used to enable load tests and to verify that the NGSA application is handling web requests correctly. More information about running WebV interactively from the command line can be found [here](https://github.com/microsoft/webvalidate).
+
+The WebV application needs to be deployed on our AKS cluster. Create a Kubernetes manifest file (yaml) to define and create the appropriate resource(s) needed to deploy WebV. The inputs needed to deploy WebV on our AKS cluster are shown below. Using these inputs, WebV will generate approximately 1 request per second when deployed.
+
+```yaml
+
+namespace:
+    ngsa
+args:
+    --sleep 1000 --run-loop --server https://< your subdomain name >.aks-sb.com --files memory-benchmark.json
+
+```
+
+### Prerequisite
+
+This challenge depends on [Deploying from Azure Container Registry](../azure-container-registry/README.md)
+
+### Validate
+
```

Comment:
they shouldn't "apply" the yaml - flux should deploy

Reply:
Updated